### PR TITLE
New Zabbix service module

### PR DESCRIPTION
**Pull request by emriver**
##### SUMMARY

New module to handle Zabbix services with Ansible.

##### ISSUE TYPE

- New Module Pull Request

##### COMPONENT NAME

zabbix_service

##### ADDITIONAL INFORMATION

Tested on Zabbix 4.0

```paste below

```

### CODE REVIEW

File: lib/ansible/modules/monitoring/zabbix/zabbix_service.py [hunk 1]
```diff
@@ -19,42 +19,49 @@
...
+        type: float
     calculate_sla:
         description:
-            - Calculate or nor the SLA value for this service, showsla in Zabbix API.
+            - If yes, calculate the SLA value for this service, showsla in Zabbix API
         required: false
+        type: bool
     algorithm:
         description:
             - ALigorithm used to calculate the sla
```
Comment (equelin) [inline lib/ansible/modules/monitoring/zabbix/zabbix_service.py]: Typo -> Algorithm

File: lib/ansible/modules/monitoring/zabbix/zabbix_service.py [hunk 2]
```diff
@@ -72,11 +79,26 @@
...
     login_user: username
     login_password: password
     name: apache2 availability
     state: dump
+
+# Creates a new Zabbix service
+- name: Manage services
+  local_action:
+        module: zabbix_service
+        server_url: "https://192.168.1.1
```
Comment (equelin) [inline lib/ansible/modules/monitoring/zabbix/zabbix_service.py]: Unnecessary or missing "

File: lib/ansible/modules/monitoring/zabbix/zabbix_service.py [hunk 3]
```diff
@@ -129,42 +153,39 @@ def delete_service(self, service_ids):
         self._zapi.service.delete(service_ids)
 
     def dump_services(self, service_ids):
-        services = self._zapi.service.get({'output': 'extend', 'filter' : {'serviceid' : service_ids}})
+        services = self._zapi.service.get({'output': 'extend', 'filter': {'serviceid': service_ids}, 'selectParent': '1'})
         return services
 
     def generate_service_config(self, name, parent, sla, calculate_sla, trigger_name, trigger_host, algorithm):
```
Comment (equelin) [inline lib/ansible/modules/monitoring/zabbix/zabbix_service.py]: def generate_service_config(self, name, parent, sla, calculate_sla=0, trigger_name, trigger_host, algorithm):

This simplify a little the code later

File: lib/ansible/modules/monitoring/zabbix/zabbix_service.py (hunk 1 above)
Comment (emriver) [reply lib/ansible/modules/monitoring/zabbix/zabbix_service.py]: fixed

File: lib/ansible/modules/monitoring/zabbix/zabbix_service.py (hunk 2 above)
Comment (emriver) [reply lib/ansible/modules/monitoring/zabbix/zabbix_service.py]: fixed

**Review by equelin — APPROVED** [review]

**Pull request merged**